xc/xc7/toolchain_wrappers/symbiflow_write_bitstream

```diff
@@ -7,19 +7,21 @@ MYPATH=`dirname ${MYPATH}`
 source ${MYPATH}/env
 echo "Writing bitstream ..."
 
-if [ -z ${XRAY_FASM2FRAMES} ]; then
-	echo "ERROR: XRAY_FASM2FRAMES variable is unset."
+DATABASE_DIR=""
+if [ ! -z ${XRAY_DATABASE_DIR} ]; then
+	DATABASE_DIR=${XRAY_DATABASE_DIR}
+elif [ ! -z ${CONDA_PREFIX} ]; then
```

Comment:
This is wrong

Reply:
You mean relying on CONDA_PREFIX? This is one of the ways we can avoid the user to set the DATABASE path when using the toolchain, unless we get to integrate the db within the tarball.

Another solution would be to get a conda package that installs the toolchain wrappers as well as the db. This way we can set a default path to the DB that can get overriden by the XRAY_DATABASE_DIR var

Reply:
We should probably provide something akin to "yosys-config" on the path, and then you could do something like:
```
DATABASE_DIR?=$(prjxray-config --database_dir)
```

Rather than depending on conda.  Then it will work under any packaging scheme, as long as `prjxray-config` is on the path.

Reply:
Right, this seems a very clean solution, so we can make the wrappers agnostic to the packaging system.

I have opened a PR in conda-packages that provides a script that has the only purpose to print out the database location: https://github.com/SymbiFlow/conda-packages/pull/121

Reply:
Configuration files are easier to work with (e.g. in `/usr/share/prjxray`) than a `yosys-config` style binary, but I can work with it if it's a script. Environment variables work well too.

I've gone as far as patching `yosys` with `bbe` to change its hardcoded data directory to avoid recompilation just to locate plugins, but I don't like that.

Reply:
The idea is that each set of artifacts generated from an independent source should be relocatable. The fact that Yosys and its plugins can be in separate repos causes trouble.